pkg/config/location.go

```diff
@@ -64,11 +64,18 @@ func configLocations() []string {
 // not set
 func PwStoreDir(mount string) string {
 	if mount != "" {
-		return fsutil.CleanPath(filepath.Join(Homedir(), ".password-store-"+strings.Replace(mount, string(filepath.Separator), "-", -1)))
+		cleanName := strings.Replace(mount, string(filepath.Separator), "-", -1)
+		// comply with the freedesktop XDG spec
+		// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+		if xdh := os.Getenv("XDG_DATA_HOME"); xdh != "" {
+			return fsutil.CleanPath(filepath.Join(xdh, "gopass", "stores", cleanName))
+		}
+		return fsutil.CleanPath(filepath.Join(Homedir(), ".local", "share", "gopass", "stores", cleanName))
```

Comment:
For better cross-platform support, maybe we can use a library to determine the correct location: https://github.com/ProtonMail/go-appdir

This should also make most uses of [mitchellh/go-homedir](https://github.com/mitchellh/go-homedir) obsolete here?

Reply:
Yes, that is a good idea. The code gets cleaner that way.
Unfortunately we can't get rid completey of `mitchellh/go-homedir`, yet.